src/Chapter2/Neven.hs

```diff
@@ -0,0 +1,5 @@
+module Chapter2.Neven where
+import Data.List
+neven n = map (\n -> 2*n) [1..n]
+
+neven2 n = foldl (\evenList n -> (evenList ++ [n])) [] [1..n]
```

Comment:
is this correct?

also, type signatures?

Reply:
output wise yea. to generate N even numbers i take a list from 1 to n and simply double it.